### PR TITLE
WS-036: Fix admin context

### PR DESCRIPTION
**Pull request by rachelledupuis**
Edit LoginManager so that admin status no longer defaults to true on login - only true if user role = admin.

### CODE REVIEW

File: frontend/src/components/login/LoginManager.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ type LoginContextType = {
   isAdmin: boolean;
   username: string;
   password: string;
-  logIn: (username: string, password: string) => void;
+  logIn: (username: string, password: string, idAdmin: boolean) => void;
```
Comment (AndrewYHuang) [inline frontend/src/components/login/LoginManager.tsx:8]: `idAdmin` typo?

File: frontend/src/clients/apiClients.ts [hunk 2]
```diff
@@ -81,6 +85,12 @@ export const login = async (
   }
 };
 
+export const isAdmin = async (username: string): Promise<boolean> => {
+  const response = await fetch(`https://localhost:5001/users/${username}`);
+  const user = await response.json();
+  return user.role === 0 ? false : true;
```
Comment (AndrewYHuang) [inline frontend/src/clients/apiClients.ts:91]: Every ternary that returns false or true can be simplified!

**Review by AndrewYHuang — DISMISSED** [review]
Couple of small comments :) N.B that this should be merged in after WS-021 #35

File: frontend/src/components/login/LoginManager.tsx (hunk 1 above)
Comment (rachelledupuis) [reply frontend/src/components/login/LoginManager.tsx:8]: Fixed

File: frontend/src/clients/apiClients.ts (hunk 2 above)
Comment (rachelledupuis) [reply frontend/src/clients/apiClients.ts]: Refactored

**Review by TimLeach635 — APPROVED** [review]
Good to go!

**Pull request merged**